tags/raw.md

```diff
@@ -0,0 +1,22 @@
+---
+title: Raw
+description: An overview of raw tags in the Liquid template language.
+---
+
+Raw temporarily disables tag processing. This is useful for generating content
```

Comment:
Good job here — getting Liquid to render Liquid tags is one of the trickiest things we have to deal with in this documentation. :)